### ISSUE TITLE
Huawei or Marshmallow Skill Issue

### ISSUE THREAD

**Issue by C10udburst** [recloudstream/cloudstream#6]
```kotlin
dex2oat (15479): Failed to create oat file: /data/dalvik-cache/arm64/data@user@0@com.lagradost.cloudstream3.debug@files@Extensions@httpsraw.githubusercontent.comrecloudstreamcloudstream-extensionsbuildsplugins.json.521180176@AkwamProvider.1992910698.cs3@classes.dex: Permission denied
art     (14406): Failed execv(/system/bin/dex2oat --runtime-arg -classpath --runtime-arg  --debuggable --instruction-set=arm64 --instruction-set-features=smp,a53 --runtime-arg -Xrelocate --boot-image=/system/framework/boot.art --runtime-arg -Xms64m --runtime-arg -Xmx512m --instruction-set-variant=generic --instruction-set-features=default --dex-file=/data/user/0/com.lagradost.cloudstream3.debug/files/Extensions/httpsraw.githubusercontent.comrecloudstreamcloudstream-extensionsbuildsplugins.json.521180176/AkwamProvider.1992910698.cs3 --oat-file=/data/dalvik-cache/arm64/data@user@0@com.lagradost.cloudstream3.debug@files@Extensions@httpsraw.githubusercontent.comrecloudstreamcloudstream-extensionsbuildsplugins.json.521180176@AkwamProvider.1992910698.cs3@classes.dex) because non-0 exit status
art     (14406): Failed to open dex file '/data/user/0/com.lagradost.cloudstream3.debug/files/Extensions/httpsraw.githubusercontent.comrecloudstreamcloudstream-extensionsbuildsplugins.json.521180176/AkwamProvider.1992910698.cs3' from memory: Unrecognized version number in /data/user/0/com.lagradost.cloudstream3.debug/files/Extensions/httpsraw.githubusercontent.comrecloudstreamcloudstream-extensionsbuildsplugins.json.521180176/AkwamProvider.1992910698.cs3: 0 3 7 
```

**Comment by C10udburst** [comment]
https://stackoverflow.com/questions/41223502/failed-to-open-dex-file-unrecognized-version-number

**Comment by C10udburst** [comment]
fixed in recloudstream/gradle@3b9c8706d9dab90d91d2ddea8cea181ed1ab9bf8
